optimum/intel/openvino/modeling_seq2seq.py

```diff
@@ -832,26 +850,25 @@ class OVModelForSpeechSeq2Seq(OVModelForSeq2SeqLM):
 
     def prepare_inputs_for_generation(
         self,
-        input_ids,
-        attention_mask: Optional[torch.LongTensor] = None,
-        decoder_attention_mask: Optional[torch.BoolTensor] = None,
+        decoder_input_ids,
         past_key_values=None,
+        attention_mask=None,
         head_mask=None,
         decoder_head_mask=None,
         cross_attn_head_mask=None,
         use_cache=None,
         encoder_outputs=None,
         **kwargs,
-    ) -> Dict:
-        if decoder_attention_mask is None:
-            decoder_attention_mask = torch.ones_like(input_ids).to(input_ids.device)
```

Comment:
could this be an issue for model that were exported with `decoder_attention_mask` ?

Reply:
done